### PR TITLE
Support alter foregin data wrapper for postgresql

### PR DESCRIPTION
**Pull request by tuichenchuxin**
ref #14015.

Changes proposed in this pull request:
- Support alter foregin data wrapper for postgresql
- add test case
- remove unsupported cases.

### CODE REVIEW

File: shardingsphere-test/shardingsphere-parser-test/src/main/resources/case/ddl/alter-foreign-data-wrapper.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  ~ limitations under the License.
+  -->
+
+<sql-parser-test-cases>
+    <alter-foreign-data-wrapper sql-case-id="alter_foreign_data_wrapper_case1" />
+    <alter-foreign-data-wrapper sql-case-id="alter_foreign_data_wrapper_case2" />
+    <alter-foreign-data-wrapper sql-case-id="alter_foreign_data_wrapper_case3" />
+    <alter-foreign-data-wrapper sql-case-id="alter_foreign_data_wrapper_case4" />
+    <alter-foreign-data-wrapper sql-case-id="alter_foreign_data_wrapper_case5" />
+    <alter-foreign-data-wrapper sql-case-id="alter_foreign_data_wrapper_case6" />
```
Comment (strongduanmu) [inline shardingsphere-test/shardingsphere-parser-test/src/main/resources/case/ddl/alter-foreign-data-wrapper.xml:25]: @tuichenchuxin Can we rename `sql-case-id` with more meaningful name?

**Review by strongduanmu — CHANGES_REQUESTED** [review]

File: shardingsphere-test/shardingsphere-parser-test/src/main/resources/case/ddl/alter-foreign-data-wrapper.xml (hunk 1 above)
Comment (tuichenchuxin) [reply shardingsphere-test/shardingsphere-parser-test/src/main/resources/case/ddl/alter-foreign-data-wrapper.xml]: ok,done

**Comment by strongduanmu** [comment]
@tuichenchuxin Please solve code conflict.

**Comment by tuichenchuxin** [comment]
> @tuichenchuxin Please solve code conflict.

Done.

**Review by strongduanmu — APPROVED** [review]
@tuichenchuxin Nice job, thank you.

**Pull request merged**